lib/private/httphelper.php

```
@@ -83,6 +83,12 @@ public function getUrlContent($url) {
 				curl_setopt($curl, CURLOPT_FOLLOWLOCATION, true);
 				curl_setopt($curl, CURLOPT_MAXREDIRS, $max_redirects);
 				$data = curl_exec($curl);
+				$code = curl_getinfo($curl, CURLINFO_HTTP_CODE);
+				if ($code != 200) {
```

> As this is a generic helper changing this here is very likely to introduce other problems.

we could throw an exception - yes more radical, but better then working with 'false'